plugins/init/Drupal.civi-setup.php

```diff
@@ -31,7 +31,7 @@
     // Compute settingsPath.
     $drupalSystem = new CRM_Utils_System_Drupal();
     $cmsPath = $drupalSystem->cmsRootPath();
-    $siteDir = _drupal_civisetup_getSiteDir($cmsPath, $_SERVER['SCRIPT_FILENAME']);
+    $siteDir = \Civi\Setup\FileUtil::getDrupalSiteDir($cmsPath);
```

Comment:
Moving `_drupal_civisetup_getSiteDir()` to a shareable class seems like a good idea.

The functionality here is really specific to the Drupal file structure, so it'd be better to call the util `\Civi\Setup\DrupalUtil::getSiteDir(...)`.